Assets/Scripts/KeyBindings/StoreKeyBindings.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+using Newtonsoft.Json;
+
+
+public static class StoreKeyBindings
+{
+    private const string custom_path = "keybindings_custom.json";
+    private const string default_path = "keybindings_default.json";
+
+    static StoreKeyBindings()
+    {
+      
+    }
+        
+    public static void SaveKeyBindings()
+    {
+        Dictionary<String, KeyCode> bindings = KeyBindings.Get();
```

Comment:
This call then then also be omitted when the keybindings are moved into here.
Note: You can then use the empty static initializer to load the config file into the dictionary.

Reply:
see first comment